src/compiler/grammar/shexl.g4

```diff
@@ -1,53 +1,157 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Weso Research Group, University of Oviedo.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+*/
+
 grammar shexl;
 
-file
- : definitions+ EOF
+// The top level element of the grammar is the shema, that is
+// what we want to represent with this grammar.
+// A schema is defined as a non-empty set of statements.
+schema
+ : statement+ EOF
+ ;
+
+// The statements are the second level of the grammar.
+// Each statement can be:
+//  - a declaration.
+//  - an import statement. import is not a definition but yes an statement as
```

Comment:
You don't need to write 'yes' in this comment.

A valid sentence would be: "- an import statement. import is not a definition but an statement as it does not declare anything."